crates/wasmcloud-host/src/host.rs

```diff
@@ -215,6 +219,7 @@ pub struct Host {
     allow_live_updates: bool,
     lattice_cache_provider_ref: Option<String>,
     strict_update_check: bool,
+    started: Arc<RwLock<bool>>,
```

Comment:
Is there a reason why this `started` variable is an `Arc<RwLock<bool>>` rather than just a `bool`? I see the benefit of ensuring multiple resources don't modify it at the same time, but it's really only touched when ensuring it has started and appears that a simple `bool` would suffice.

Reply:
because the host, by rule, doesn't have any `mut` methods. Mutation has to be kept inside. The host also has requirements by consumers that it be placeable inside refcells, which means it can't do anything internally that isn't threadsafe. So, hiding the mutation of the started field inside an rwlock is the only way to go. Though, we can actually remove the `Arc`.